e2e/various-suite/loki-query-builder.spec.ts

```diff
@@ -14,6 +14,16 @@ const addDataSource = () => {
   });
 };
 
+// We need this because the final query is split into separate DOM elements using e2e().contains(finalQuery).should('not.exist'); does not detect the query.
+const checkFinalQuery = (query: 'be.visible' | 'not.exist') => {
+  const keywords = ['rate', 'instance1|instance2', 'logfmt', '__error__', '$__interval'];
+  for (const word of keywords) {
+    e2e().contains(word).should(query);
+  }
+};
```

Comment:
Looks like `checkFinalQuery` is just called once, and with `'be.visible'`?

```suggestion
const checkFinalQuery = () => {
  const keywords = ['rate', 'instance1|instance2', 'logfmt', '__error__', '$__interval'];
  for (const word of keywords) {
    e2e().contains(word).should('be.visible');
  }
};
```

@ivanahuckova

Reply:
This is @gwdawson 's PR, so I am going to let him get back to you on this. 🙂

Reply:
Looking forward to it 🧐

Reply:
Looking forward to it 🧐. By the way, I just realized that my mention was aimed at the other comment, not this one.

Reply:
this was initially used for both `be.visible` and `not.exist` but i guess i ended up replacing the second function call and didn't update the function.